Storing of total queue can fail with NPE

When storing total queue into `results.edn`, storing multiset can fail with following NPE:
```
ERROR in (rabbit-test) (FutureTask.java:122)
Uncaught exception, not in assertion.
expected: nil
  actual: java.util.concurrent.ExecutionException: java.lang.NullPointerException
 at java.util.concurrent.FutureTask.report (FutureTask.java:122)
    java.util.concurrent.FutureTask.get (FutureTask.java:192)
    clojure.core$deref_future.invokeStatic (core.clj:2292)
    clojure.core$future_call$reify__8097.deref (core.clj:6894)
    clojure.core$deref.invokeStatic (core.clj:2312)
    clojure.core$deref.invoke (core.clj:2298)
    clojure.core$map$fn__5587.invoke (core.clj:2745)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$dorun.invokeStatic (core.clj:3125)
    clojure.core$dorun.invoke (core.clj:3125)
    jepsen.store$save_2_BANG_.invokeStatic (store.clj:384)
    jepsen.store$save_2_BANG_.invoke (store.clj:380)
    jepsen.core$analyze_BANG_.invokeStatic (core.clj:522)
    jepsen.core$analyze_BANG_.invoke (core.clj:506)
    jepsen.core$run_BANG_$fn__6009.invoke (core.clj:637)
    jepsen.core$run_BANG_.invokeStatic (core.clj:611)
    jepsen.core$run_BANG_.invoke (core.clj:539)
    jepsen.rabbitmq_test$fn__4846.invokeStatic (rabbitmq_test.clj:48)
    jepsen.rabbitmq_test/fn (rabbitmq_test.clj:47)
    clojure.test$test_var$fn__9209.invoke (test.clj:716)
    clojure.test$test_var.invokeStatic (test.clj:716)
    clojure.test$test_var.invoke (test.clj:707)
    clojure.test$test_vars$fn__9235$fn__9240.invoke (test.clj:734)
    clojure.test$default_fixture.invokeStatic (test.clj:686)
    clojure.test$default_fixture.invoke (test.clj:682)
    clojure.test$test_vars$fn__9235.invoke (test.clj:734)
    clojure.test$default_fixture.invokeStatic (test.clj:686)
    clojure.test$default_fixture.invoke (test.clj:682)
    clojure.test$test_vars.invokeStatic (test.clj:730)
    clojure.test$test_all_vars.invokeStatic (test.clj:736)
    clojure.test$test_ns.invokeStatic (test.clj:757)
    clojure.test$test_ns.invoke (test.clj:742)
    user$eval233$fn__294.invoke (form-init6434085779704025133.clj:1)
    clojure.lang.AFn.applyToHelper (AFn.java:156)
    clojure.lang.AFn.applyTo (AFn.java:144)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.core$apply.invoke (core.clj:652)
    leiningen.core.injected$compose_hooks$fn__163.doInvoke (form-init6434085779704025133.clj:1)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:657)
    clojure.core$apply.invoke (core.clj:652)
    leiningen.core.injected$run_hooks.invokeStatic (form-init6434085779704025133.clj:1)
    leiningen.core.injected$run_hooks.invoke (form-init6434085779704025133.clj:1)
    leiningen.core.injected$prepare_for_hooks$fn__168$fn__169.doInvoke (form-init6434085779704025133.clj:1)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.AFunction$1.doInvoke (AFunction.java:29)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$map$fn__5587.invoke (core.clj:2747)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.Cons.next (Cons.java:39)
    clojure.lang.RT.boundedLength (RT.java:1785)
    clojure.lang.RestFn.applyTo (RestFn.java:130)
    clojure.core$apply.invokeStatic (core.clj:659)
    clojure.test$run_tests.invokeStatic (test.clj:767)
    clojure.test$run_tests.doInvoke (test.clj:767)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:657)
    clojure.core$apply.invoke (core.clj:652)
    user$eval233$fn__306$fn__339.invoke (form-init6434085779704025133.clj:1)
    user$eval233$fn__306$fn__307.invoke (form-init6434085779704025133.clj:1)
    user$eval233$fn__306.invoke (form-init6434085779704025133.clj:1)
    user$eval233.invokeStatic (form-init6434085779704025133.clj:1)
    user$eval233.invoke (form-init6434085779704025133.clj:1)
    clojure.lang.Compiler.eval (Compiler.java:7062)
    clojure.lang.Compiler.eval (Compiler.java:7052)
    clojure.lang.Compiler.load (Compiler.java:7514)
    clojure.lang.Compiler.loadFile (Compiler.java:7452)
    clojure.main$load_script.invokeStatic (main.clj:278)
    clojure.main$init_opt.invokeStatic (main.clj:280)
    clojure.main$init_opt.invoke (main.clj:280)
    clojure.main$initialize.invokeStatic (main.clj:311)
    clojure.main$null_opt.invokeStatic (main.clj:345)
    clojure.main$null_opt.invoke (main.clj:342)
    clojure.main$main.invokeStatic (main.clj:424)
    clojure.main$main.doInvoke (main.clj:387)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:702)
    clojure.main.main (main.java:37)
Caused by: java.lang.NullPointerException: null
 at multiset.core.MultiSet.iterator (core.clj:73)
    clojure.lang.RT.iter (RT.java:566)
    clojure.core$sequence.invokeStatic (core.clj:2655)
    clojure.core$sequence.invoke (core.clj:2639)
    fipp.edn$pretty_coll.invokeStatic (edn.cljc:15)
    fipp.edn$pretty_coll.invoke (edn.cljc:8)
    fipp.edn.EdnPrinter.visit_set (edn.cljc:65)
    fipp.visit$visit_STAR_.invokeStatic (visit.cljc:48)
    fipp.visit$visit_STAR_.invoke (visit.cljc:32)
    fipp.visit$visit.invokeStatic (visit.cljc:61)
    fipp.visit$visit.invoke (visit.cljc:58)
    fipp.edn.EdnPrinter/fn (edn.cljc:62)
    fipp.edn$pretty_coll$fn__2300.invoke (edn.cljc:12)
    clojure.core$map$fn__5583$fn__5584.invoke (core.clj:2734)
    clojure.lang.TransformerIterator.step (TransformerIterator.java:79)
    clojure.lang.TransformerIterator.hasNext (TransformerIterator.java:97)
    clojure.lang.RT$4.invoke (RT.java:515)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$map$fn__5587.invoke (core.clj:2738)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$apply.invokeStatic (core.clj:652)
    clojure.core$mapcat.invokeStatic (core.clj:2775)
    clojure.core$mapcat.doInvoke (core.clj:2775)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    fipp.engine$serialize.invokeStatic (engine.cljc:15)
    fipp.engine$serialize.invoke (engine.cljc:12)
    clojure.core$map$fn__5587.invoke (core.clj:2745)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$apply.invokeStatic (core.clj:652)
    clojure.core$mapcat.invokeStatic (core.clj:2775)
    clojure.core$mapcat.doInvoke (core.clj:2775)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    fipp.engine$serialize.invokeStatic (engine.cljc:15)
    fipp.engine$serialize.invoke (engine.cljc:12)
    fipp.engine$fn__2260.invokeStatic (engine.cljc:63)
    fipp.engine/fn (engine.cljc:58)
    clojure.lang.MultiFn.invoke (MultiFn.java:229)
    fipp.engine$serialize.invokeStatic (engine.cljc:18)
    fipp.engine$serialize.invoke (engine.cljc:12)
    clojure.core$map$fn__5587.invoke (core.clj:2745)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$apply.invokeStatic (core.clj:652)
    clojure.core$mapcat.invokeStatic (core.clj:2775)
    clojure.core$mapcat.doInvoke (core.clj:2775)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    fipp.engine$serialize.invokeStatic (engine.cljc:15)
    fipp.engine$serialize.invoke (engine.cljc:12)
    fipp.engine$fn__2245.invokeStatic (engine.cljc:48)
    fipp.engine/fn (engine.cljc:47)
    clojure.lang.MultiFn.invoke (MultiFn.java:229)
    fipp.engine$serialize.invokeStatic (engine.cljc:18)
    fipp.engine$serialize.invoke (engine.cljc:12)
    clojure.core$map$fn__5587.invoke (core.clj:2745)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$apply.invokeStatic (core.clj:652)
    clojure.core$mapcat.invokeStatic (core.clj:2775)
    clojure.core$mapcat.doInvoke (core.clj:2775)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    fipp.engine$serialize.invokeStatic (engine.cljc:15)
    fipp.engine$serialize.invoke (engine.cljc:12)
    fipp.engine$fn__2229.invokeStatic (engine.cljc:35)
    fipp.engine/fn (engine.cljc:34)
    clojure.lang.MultiFn.invoke (MultiFn.java:229)
    fipp.engine$serialize.invokeStatic (engine.cljc:18)
    fipp.engine$serialize.invoke (engine.cljc:12)
    clojure.core$map$fn__5587.invoke (core.clj:2745)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$apply.invokeStatic (core.clj:652)
    clojure.core$mapcat.invokeStatic (core.clj:2775)
    clojure.core$mapcat.doInvoke (core.clj:2775)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    fipp.engine$serialize.invokeStatic (engine.cljc:15)
    fipp.engine$serialize.invoke (engine.cljc:12)
    clojure.core$map$fn__5587.invoke (core.clj:2745)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$apply.invokeStatic (core.clj:652)
    clojure.core$mapcat.invokeStatic (core.clj:2775)
    clojure.core$mapcat.doInvoke (core.clj:2775)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    fipp.engine$serialize.invokeStatic (engine.cljc:15)
    fipp.engine$serialize.invoke (engine.cljc:12)
    fipp.engine$fn__2260.invokeStatic (engine.cljc:63)
    fipp.engine/fn (engine.cljc:58)
    clojure.lang.MultiFn.invoke (MultiFn.java:229)
    fipp.engine$serialize.invokeStatic (engine.cljc:18)
    fipp.engine$serialize.invoke (engine.cljc:12)
    clojure.core$map$fn__5587.invoke (core.clj:2745)
    clojure.lang.LazySeq.sval (LazySeq.java:40)
    clojure.lang.LazySeq.seq (LazySeq.java:49)
    clojure.lang.RT.seq (RT.java:528)
    clojure.core$seq__5124.invokeStatic (core.clj:137)
    clojure.core$apply.invokeStatic (core.clj:652)
    clojure.core$mapcat.invokeStatic (core.clj:2775)
    clojure.core$mapcat.doInvoke (core.clj:2775)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    fipp.engine$serialize.invokeStatic (engine.cljc:15)
    fipp.engine$serialize.invoke (engine.cljc:12)
    fipp.engine$fn__2245.invokeStatic (engine.cljc:48)
    fipp.engine/fn (engine.cljc:47)
    clojure.lang.MultiFn.invoke (MultiFn.java:229)
    fipp.engine$serialize.invokeStatic (engine.cljc:18)
    fipp.engine$serialize.invoke (engine.cljc:12)
    fipp.engine$pprint_document.invokeStatic (engine.cljc:236)
    fipp.engine$pprint_document.invoke (engine.cljc:234)
    fipp.edn$pprint.invokeStatic (edn.cljc:100)
    fipp.edn$pprint.invoke (edn.cljc:91)
    fipp.edn$pprint.invokeStatic (edn.cljc:92)
    fipp.edn$pprint.invoke (edn.cljc:91)
    jepsen.store$write_results_BANG_.invokeStatic (store.clj:344)
    jepsen.store$write_results_BANG_.invoke (store.clj:340)
    jepsen.store$save_2_BANG_$fn__4297.invoke (store.clj:384)
    clojure.core$binding_conveyor_fn$fn__5476.invoke (core.clj:2022)
    clojure.lang.AFn.call (AFn.java:18)
    java.util.concurrent.FutureTask.run (FutureTask.java:266)
    java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1149)
    java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:624)
    java.lang.Thread.run (Thread.java:748)
```

--- a/jepsen/project.clj
+++ b/jepsen/project.clj
@@ -16,7 +16,7 @@
                  [http-kit "2.1.18"]
                  [ring "1.6.0-beta5"]
                  [hiccup "1.0.5"]
-                 [org.clojars.achim/multiset "0.1.0"]
+                 [metametadata/multiset "0.1.1"]
                  [byte-streams "0.2.2"]
                  [dom-top "1.0.4"]
                  [slingshot "0.12.2"]

